plugin.video.tagesschau/addon.xml

```diff
@@ -12,17 +12,18 @@
     <provides>video</provides>
   </extension>
   <extension point="xbmc.addon.metadata">
-    <language>de</language>
+    <language>de_DE</language>
```

Comment:
I think this should be `<language>en de</language>`. Perhaps they could have separate tags, but it should be the short 2-char ISO language value here. In the `summary` tags and so, it is correct now.